### PR TITLE
Add possibility to get info from /usr/lib/os-release file.

### PR DESCRIPTION
**Pull request by DmitriiBobreshev**
**Describtion**:
As in Linux specs, the release information might be placed not only in /etc/os-release but in /usr/lib/os-release [as well](https://www.linux.org/docs/man5/os-release.html), and the file /etc/os-release might not exist(e.g. fresh arch linux I'm my case).
The PR allows taking VERSION_ID and ID from /usr/lib/os-release if /etc/os-release is not presented. 
 Fixed failed step if the version is not provided for telemetry publishing

**Related WI**: 2048785

**Testing**:
Testing locally on Arch Linux with missing /etc/os-release link

### CODE REVIEW

File: src/Agent.Worker/TaskRunner.cs [hunk 1]
```diff
@@ -645,10 +645,17 @@ private void PublishTelemetry(Definition taskDefinition, HandlerData handlerData
                 publishTelemetryCmd.Initialize(HostContext);
                 publishTelemetryCmd.ProcessCommand(ExecutionContext, cmd);
             }
-            catch (NullReferenceException ex)
-            {
-                ExecutionContext.Debug($"ExecutionHandler telemetry wasn't published, because one of the variables is null");
-                ExecutionContext.Debug(ex.ToString());
+            catch (Exception ex)            
+            {                    
+                if (ex is FormatException || ex is ArgumentNullException || ex is NullReferenceException)
```
Comment (KonstantinTyukalov) [inline src/Agent.Worker/TaskRunner.cs:650]: nit: You can use `when` ex filter: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/statements/exception-handling-statements#a-when-exception-filter

**Review by KonstantinTyukalov — APPROVED** [review]

File: src/Agent.Worker/TaskRunner.cs (hunk 1 above)
Comment (DmitriiBobreshev) [reply src/Agent.Worker/TaskRunner.cs]: added exception filter, thanks!

**Review by kirill-ivlev — APPROVED** [review]

File: src/Agent.Sdk/Util/PlatformUtil.cs [hunk 2]
```diff
@@ -152,13 +152,33 @@ private static void DetectRHEL6()
             }
         }
 
+        private static string GetLinuxReleaseFilePath()
+        {
+            if (RunningOnLinux)
+            {
+                if (File.Exists("/etc/os-release"))
```
Comment (DergachevE) [inline src/Agent.Sdk/Util/PlatformUtil.cs:159]: Can these file paths be made as const?

Comment (DmitriiBobreshev) [reply src/Agent.Sdk/Util/PlatformUtil.cs]: Replaced with a read-only array of strings, thanks!

**Review by DergachevE — APPROVED** [review]

**Pull request merged**